R/SEIRAge.R

```diff
@@ -0,0 +1,275 @@
+#' Defines an age-structured SEIR model and solves the set of
+#' ordinary differential equations of the model with a chosen method of
+#' numerical integration.
+#'
+#' @slot name character representing name of model
```

Comment:
I think @ben18785 mentioned this before, you might want to remove the 'name' slot, it is not used in the class.